views/js/qtiCreator/widgets/component/minMax/minMax.js

```diff
@@ -47,7 +47,10 @@ define([
             //default help message
             helpMessage: __(
                 'The minimum number of choices that the candidate is required to select to form a valid response.'
-            )
+            ),
+
+            // determines if field can be null
+            canBeNull: false,
         },
```

Comment:
1. From ticket ACs, "by default, min is set to 0 and max should not be set.". Create new interaction, enable "map response", set scores for choices. `min` field in UI is unchedked+disabled and empty, but shouldn't it be checked+enabled and have `0` inside?
2. From ticket ACs, "_max should be strictly greater than min._" Now I can enter equal values: `min:18, max:18`. Is it as expected?

Current behavior is fine for me, just it contradicts ACs.

Reply:
Point `#1` should be addressed, yes.